ord_schema/units.py

```diff
@@ -138,6 +138,63 @@
     "m": "ambiguous between meter and minute",
 }
 
+_UNIT_CONVERSIONS = {
```

Comment:
Since I didn't add an exhaustive set of test cases, would appreciate another pair of eyes on this (@skearnes, @qai222)